### ISSUE TITLE
Recompile sass files when play reloads

### ISSUE THREAD

**Issue by busti** [irundaia/sbt-sassify#31]
Would it be possible to reload sass files whenever they have been changed and a http request is made to play when in interactive mode?  
Similar to play sources recompiling on reloads.

**Comment by irundaia** [comment]
@Busti, this should work out of the box. If it does not work for you, could you perhaps provide git repo that showcases this issue?

**Comment by busti** [comment]
It only works for the first reload for me. After that changes have no effect anymore.  
I created a repo in an attempt to reproduce the issue, but I was unable to do so just yet.  
This is the commit that adds sbt-sassify:
https://github.com/Busti/play-silhouette-seed/commit/708263313e65d7741478b03d6a53fa9e0c30ec1b

I chose mohiva/play-slick-seed since it is closes to what I am working with.  
I recently migrated to Play 2.6 and did not have the issue before. Play also does not print it's pink `--- (RELOAD) ---` message in my project (very rarely it does). What that shows me is that I probably missed something while migrating. I guess I will have to go through all the files once more.  

Thanks for your quick reply though ❤
